packages/fuel-gauge/src/predicate/predicate-arguments.test.ts

```diff
@@ -215,14 +230,18 @@ describe('Predicate', () => {
     });
 
     it('calls a predicate with valid multiple arguments and returns true', async () => {
-      const predicate = new Predicate(predicateBytesMulti, provider, predicateAbiMulti);
+      const predicate = new Predicate({
+        bytecode: predicateBytesMulti,
+        abi: predicateAbiMulti,
+        provider,
+        inputData: [20, 30],
+      });
 
       const initialPredicateBalance = await fundPredicate(wallet, predicate, amountToPredicate);
       const initialReceiverBalance = await receiver.getBalance();
 
       // #region predicate-multi-args
-      // #context const predicate = new Predicate(bytecode, chainId, abi);
-      predicate.setData(20, 30);
+      // #context const predicate = new Predicate(bytecode, chainId, abi, [20, 30]);
```

Comment:
This appears to be the incorrect signature for `Predicate` :)

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1826/commits/4f921ed92fb41eb7813b02d4dd1d4fd6d4977c36